source/agora/consensus/EnrollmentManager.d

```diff
@@ -847,7 +859,6 @@ unittest
     assert(man.getEnrolledHeight(utxo_hash2) == 0);
     man.getUnregistered(enrolls);
     assert(enrolls.length == 1);
-    assert(man.count() == 2);  // has not changed
```

Comment:
Why was the unittest removed instead of just changing the number that's tested?

Reply:
> Why was the unittest removed instead of just changing the number that's tested?

This kind of deletion of code could be confusing. I'll restore it.